pkg/exasol-rest-api/response_types.go

```diff
@@ -0,0 +1,20 @@
+package exasol_rest_api
+
+import "encoding/json"
+
+type BaseResponse struct {
+	Status       string          `json:"status"`
```

Comment:
Is this the http response status? If yes this should be an integer.

Reply:
No, it's a mapping of WebSockets response format, the status is "ok"/"error" and maybe something else